### ISSUE TITLE
[Feature] When using a different shell than explorer allow minimize on boot

### ISSUE THREAD

**Issue by Sp3ci3s8472** [Ryochan7/DS4Windows#2577]
I have a HTPC running Windows 10 and Kodi; to speed up the boot and really create that media-pc feel I have set Kodi as the shell. However, when I try to start DS4Windows minimized this will result in a crash of DS4Windows because it can't detect the task area I assume. It also might be that because Kodi is running as a shell it can't reliably detect the user session?
It would be nice if there is a way to fix that.

The current workaround I now use is start DS4Windows as a task and not start it minimized. Kodi boots slower than DS4Windows so that is in the forefront.

I attached the logfile:
[ds4windows_log_20220905.12.txt](https://github.com/Ryochan7/DS4Windows/files/9535384/ds4windows_log_20220905.12.txt)

**Comment by Ryochan7** [comment]
Doesn't the taskbar and system tray require Explorer to be running to even function? Just checked and that is the case. This issue does not make sense. You would likely need to use the **Minimize to Taskbar** setting for that configuration. That option seems to work fine after Explorer has been ended.

**Comment by Sp3ci3s8472** [comment]
You are correct but that is not my point. I don't want the application to stop working correctly when I minimize it when there is no taskbar or system tray.
In my testing the application seems to run in the background when you minimize it at boot or manually. The only problem is when I connect controllers or have controllers connected and I minimized the application it stops working correctly. So basically it is useless at that point.
Tomorrow I could test it again if you like?

Did you manually stop Explorer to stop that or did you switch the shell? Windows 10 and probably 11 while they say they use explorer as shell it is a different service. E.g. if I set Kodi as a shell and try to start explorer (after killing Kodi) I will never get the taskbar back. This behaviour is different since the first release of Windows 10 where it behaved like Windows 7 and 8.

**Comment by johnodon** [comment]
I came here to say that I am in exactly the same situation, however, my shell is EmulationStation-DE (https://es-de.org).  I will be watching this thread closely.

John
